src/removeTask.js

```diff
@@ -1,6 +1,13 @@
-const removeTask = (id, tasks) => {
-  const newTasks = tasks.filter((task) => task.id !== id);
-  return newTasks;
+const removeTask = (taskId, tasks) => {
```

Comment:
- [ ] As the previous reviewer suggested, the task index should be updated, based on your list example. 


I have this:

```javascript
id:1 name: work
id:2 name: coding
id:3 name: walking
``` 
If I delete coding from list the remaining should be like this:

```javascript
id:1 name: work
id:2 name: walking
``` 
Also If I want to add new task the list should be like this:

```javascript
id:1 name: work
id:2 name: walking
id:3 name: new task
```